src/events.js

```diff
@@ -1,194 +1,191 @@
 export default [
   {
-    title: "II Hackathon Kognitywistyczny Cog_HACK",
+    title: 'II Hackathon Kognitywistyczny Cog_HACK',
```

Comment:
It's k, `prettier` for the win! Is that `airbnb` config?

Reply:
It is, sorry if semi colons disturbs. Even if not, they makes commits trashy a bit.

Reply:
If it comes to that I agree. Feel free to revert changes in that file using `git checkout HEAD~10 -- ./src/events.js` (you may need to adjust path to mentioned file).

On the other hand I'd be pretty happy with introducing consistent formatting and prettier config for that file and extend github action with additional check if formating is preserved - I'll make an issue for that 😅

Reply:
Just installed and added to readme